code/src/App.js

```diff
@@ -1,9 +1,74 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { ThoughtForm } from 'components/ThoughtForm';
+import { ThoughtList } from 'components/ThoughtList';
 
 export const App = () => {
+  const [thoughtList, setThoughtList] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [newThought, setNewThought] = useState('');
+
+  // Function to get the thoughts currently in the API:
+  const fetchThoughts = () => {
+    setLoading(true);
+    fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts')
+      .then((res) => res.json())
```

Comment:
Cleare and neat code throughout the app.js. Very easy to follow with your comments, love that.